lib/brainstem/dsl/base_block.rb

```diff
@@ -22,8 +22,23 @@ def descend(klass, new_config = configuration, new_options = {}, &block)
 
         def parse_args(args)
           options = args.last.is_a?(Hash) ? args.pop : {}
-          description = args.shift
-          [description, options]
+          maybe_description = args.shift
+
+          if maybe_description.is_a?(Hash)
+            options = options.merge(maybe_description)
```

Comment:
I think it's if using the old (non hash) syntax. In that case, `options` will be an empty Hash (see line 24 of this file.